StPetersburg/RNBefns24.xml

```diff
@@ -0,0 +1,181 @@
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0508RNB"/>
+                        <idno>RNB Ef. n.s. 24</idno>
+                        <altIdentifier>
+                           <idno>RNB 1900, ruk. 140</idno>
+                        </altIdentifier>
+                        <altIdentifier>
+                           <idno>Turaev cat. I.28</idno>
+                        </altIdentifier>
+                              </msIdentifier>
+               <msContents>
+
+               </msContents>
+
+               <physDesc>
+                  <objectDesc form="Leaf">
+                     <supportDesc>
+                        <support>
+                           <material key="paper"/>
+                           <material key="cardboard"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf">1</measure>
+                           <dimensions type="outer" unit="mm">
+                              <height>300</height>
+                              <width>645</width>
+                              <depth/>
+                           </dimensions>
+                        </extent>
+                        <condition key="good"/>
+                     </supportDesc>
+
+            </objectDesc>
+
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <date notBefore="1896" notAfter="1900"/>
+                        </handNote>
+                  </handDesc>
+
+                  <decoDesc>
+                     <decoNote type="drawing" xml:id="d1">
+                        <locus target="#1r"/>
+                        <desc>Drawing is made with ink, and is partly colored.
+                          Emperor <persName ref="PRS7006Menilek">
+                          Mǝnilǝk II</persName> riding on the horse,
+                          accompanied by servants and soldiers with guns and banners.
+                          The caption in the bottom right corner reads
+                          <foreign xml:lang="am">ጃንሆይ፡ ያንድ᎓ ቀን᎓ አይን᎓ ነውና᎓ እንዳይነውርብኝ᎓
+                          የየካቲት᎓ ጊወርጊስን᎓ ያነገሱለት᎓ አይቼ᎓ ዛሬ᎓ ሰራሁት᎓ </foreign>
+                        <foreign xml:lang="en">Your Majesty, this is an impression of one day; may it
+                          not become a shame for me.
+                          Today I have made (a drawing of) what
+                          I saw when you enthroned (the <foreign>tābot</foreign>
+                          of) <persName ref="PRS4585Georgeo">George</persName> in <date>Yakkātit</date>.
+                        </foreign>
+                          </desc>
+                        <note>The drawing refers to <persName ref="PRS7006Menilek">
+                          Mǝnilǝk II</persName> attending the feast of introducing
+                          a <persName ref="PRS4585Georgeo" type="tabot">George of Lydda</persName>
```

Comment:
I'm not sure whether this is the correct mark up to refer to a tabot. We have used it only in the place records. @PietroLiuzzo ?

Reply:
ok